app/login/request_ip_result.php

```diff
@@ -23,21 +23,51 @@
 	# strip input tags
 	$_POST = $Admin->strip_input_tags($_POST);
 
+	# verify MAC Address
+	if(!$Tools->validate_mac($_POST['mac']) ) 		{ $Result->show("danger", _('Please provide valid mac address').'! ('._('mac').': '.$Tools->strip_xss($_POST['mac']).')', true); }
```

Comment:
Hello,
Making MAC mandatory on all IP requests may not be right for everyone. 

I think we need a way of configuring which fields are mandatory.

Reply:
Hello
Thanks, you are right ! would be useful. All the custom fields already have it.

Reply:
Hi,
Because both may have different fields... example on the request the desired date to get IP approved. Furthermore during the approval of the request more technical data could be documented that will never be known by the request user and included it on the request will make request form too long.